docs/ChangeLog/20191129/PR7008.md

```diff
@@ -0,0 +1,4 @@
+* Moving backlight keycode handling to `process_keycode/`
```

Comment:
Would you please move/rename this file to `docs/ChangeLog/20200229/PR7008.md`?

Do that and I'll merge this into `future` branch.